### DISCUSSION TITLE
OpenAPI block; deactivating samples seems not working

### DISCUSSION THREAD

**Discussion by rocchisanijl** [GitbookIO/community#568] [4. Open API] [answered]
### Ask a question, start a conversation, or make an annoucement

I tried to deactivate samples regarding openapi blocks but it seems not working

[swagger_e-financing_1.4.0_uatc.zip](https://github.com/GitbookIO/community/files/14883415/swagger_e-financing_1.4.0_uatc.zip)

see extract code YAML

`openapi: 3.0.3
info:
  title: E-financing
  description: >
    The E-financement API is used **to initiate and manage a E-financement
 ...
  x-sg-name: e-financing
  x-irt-code: A8821
  x-trigram-code: BAX
  x-api-audience: PUBLIC
  termsOfService: |
    24x7
x-hideTryItPanel: true
x-codeSamples: false
servers:
  - url: >-
      https://api.scalexpert.uatc.societegenerale.com/baas/uatc/e-financing/api/v1
tags:
  - name: Eligibility
  - name: Solution simulations
  - name: Subscriptions
  - name: Delivery
  - name: Cancellation
paths:`

**Comment by taszap** [comment]
Hi @rocchisanijl thanks for reaching out to us, could you please contact support@gitbook.com with the name of the space it happens it? We will investigate.

**Reply by addisonschultz** [reply]
Is this still happening @rocchisanijl ? This should be working

**Reply by rocchisanijl** [reply, marked as answer]
Yes it seems to work now thanks
